ticdc/ticdc-sink-to-mysql.md

```diff
@@ -105,8 +105,8 @@ level = "eventual"
 # 单个 redo log 文件大小，单位 MiB，默认值 64，建议该值不超过 128。
 max-log-size = 64
 
-# 刷新或上传 redo log 至 S3 的间隔，单位毫秒，默认 1000，建议范围 500-2000。
-flush-interval = 1000
+# 刷新或上传 redo log 至 S3 的间隔，单位毫秒，建议该参数 >= 2000。
+flush-interval = 2000
```

Comment:
Is this config the same as `flush-interval` in [Replicate Data to Storage Services](/ticdc/ticdc-sink-to-cloud-storage.md#optional-parameters)?
![image](https://user-images.githubusercontent.com/89768198/211237045-17f569d9-d866-49e2-98b2-54cb874678f1.png)

Reply:
No, this is configuration in sink module. And we fix the configuration of redo module in this pr.